lib/components/inputs/inputview.coffee

```diff
@@ -449,14 +451,18 @@ module.exports = class KDInputView extends KDView
 
     # input content is copied into clone
     # element to get calculated height
-    @_clone = $ '<div/>', class : 'invisible'
+    @_clone = $ '<div/>', { class: 'invisible' }
+
+    { type } = @getOptions()
+    isVertical = type.toLowerCase() is 'textarea'
```

Comment:
operation we are making is `isVertical` but the option we are passing is `textarea`. Can't an input autogrow both vertically and horizontally?